nucypher/characters/lawful.py

```diff
@@ -1363,7 +1385,7 @@ def encrypt_message():
 
             response_data = {
                 'result': {
-                    'message_kit': b64encode(message_kit.to_bytes()).decode(),   # FIXME
+                    'message_kit': b64encode(message_kit.to_bytes()).decode(),  # FIXME
```

Comment:
Fix what?

Reply:
I mean, do you see that line?  :-)

The `FIXME` isn't new; @KPrasch put it there in Denver while we were preparing for me to need to type this ridiculous logic into a terminal live on stage.